src/locale/fr_FR.ts

```diff
@@ -0,0 +1,344 @@
+    gammaValueLabel: 'Transparence (Gamma)',
+    colorMapLabel: 'Color Map',
+    symbolizerLabel: 'Symbolisation',
+    channelSelectionLabel: 'Sélection des canaux'
+  },
+  GsRasterChannelEditor: {
+    channelSelectionLabel: 'Modifier les canaux',
+    redBandLabel: 'Rouge',
+    greenBandLabel: 'Vert',
+    blueBandLabel: 'Bleu',
+    grayBandLabel: 'Gris',
+    channelSelectionGrayLabel: 'Gris',
+    channelSelectionRgbLabel: 'RVB',
+    titleLabel: 'Sélection des canaux'
+  },
+  GsColorMapEditor: {
+    typeLabel: 'Type',
+    extendedLabel: 'Profondeur de couleur',
+    colorMapEntriesLabel: 'Colormap',
+    titleLabel: 'Color Map',
+    nrOfClassesLabel: 'Nombre de classes',
+    colorRampLabel: 'Palette de couleurs',
+    colorLabel: 'Couleur',
+    quantityLabel: 'Quantité',
+    labelLabel: 'Étiquette',
+    opacityLabel: 'Opacité'
+  },
+  GsPreview: {
+    openEditorText: 'Modifier la symbolisation',
+    closeEditorText: 'Fermer l\'éditeur'
+  },
+  GsColorField: {
+    closeText: 'Fermer',
+    editText: 'Modifier',
+    chooseText: 'Choisir',
+  },
+  GsKindField: {
+    symbolizerKinds: {
+      Mark: 'Marqueur',
+      Fill: 'Remplissage',
+      Icon: 'Icône',
+      Line: 'Ligne',
+      Text: 'Texte',
+      Raster: 'Image'
+    }
+  },
+  GsGraphicTypeField: {
+    Mark: 'Marqueur',
+    Icon: 'Icône'
+  },
+  GsRgbChannelField: {
+    redLabel: 'Canal rouge',
+    greenLabel: 'Canal vert',
+    blueLabel: 'Canal bleu'
+  },
+  GsGrayChannelField: {
+    grayLabel: 'Canal gris'
+  },
+  GsScaleDenominator: {
+    minScaleDenominatorLabelText: 'Échelle minimum',
+    maxScaleDenominatorLabelText: 'Échelle maximum',
+    minScaleDenominatorPlaceholderText: 'Saisir une échelle minimum (Facultatif)',
+    maxScaleDenominatorPlaceholderText: 'Saisir une échelle maximum (Facultatif)'
+  },
+  GsWellKnownNameField: {
+    wellKnownNames: {
+      Circle: 'Cercle',
+      Square: 'Carré',
+      Triangle: 'Triangle',
+      Star: 'Etoile',
+      Cross: 'Croix',
+      X: 'X'
+    }
+  },
+  GsColorMapEntryField: {
+    colorLabel: 'Couleur',
+    labelLabel: 'Étiquette de légende',
+    quantityLabel: 'Quantité',
+    opacityLabel: 'Opacité'
+  },
+  GsChannelField: {
+    sourceChannelNameLabel: 'Nom du canal',
+    contrastEnhancementTypeLabel: 'Amélioration du contraste',
+    gammaValueLabel: 'Transparence (Gamma)'
+  },
+  GsColorMapTypeField: {
+    rampMapTypeLabel: 'Interpolé',
+    intervalsMapTypeLabel: 'Intervalles',
+    valuesMapTypeLabel: 'Valeurs'
+  },
+  GsSymbolizerEditor: {
+    kindFieldLabel: 'Type'
+  },
+  GsSymbolizerEditorWindow: {
+    symbolizersEditor: 'Éditeur de symbolisation'
+  },
+  GsFilterEditorWindow: {
+    filterEditor: 'Éditeur de filtre'
+  },
+  GsMultiEditor: {
+    add: 'Ajouter',
+    remove: 'Supprimer'
+  },
+  GsUploadButton: {
+    upload: 'Téléverser'
+  },
+  GsFilterTree: {
+    andDrpdwnLabel: 'AND-Filter',
+    orDrpdwnLabel: 'OR-Filter',
+    notDrpdwnLabel: 'NOT-Filter',
```

Comment:
Is it intentional to use the `-Filter` suffix here instead of something like `-filtre`?

Reply:
I was unsure of how I would like it to appear in the UI
So there are still some strings in english, but not much.

Reply:
Alright, no problem. This is definitely unblocking for me